File: 283-move-zeroes/283-move-zeroes.c


void moveZeroes(int* nums, int numsSize){
int count = 0;
        for(int i = 0; i<numsSize;i++){
            if(nums[i]!=0){
                int temp  = nums[count];
                nums[count] = nums[i];
                nums[i] = temp;
                count++;
            }
        }
}

File: 283-move-zeroes/283-move-zeroes.py
class Solution(object):
    def moveZeroes(self, nums):
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        count = 0
        for i in range(len(nums)):
            if(nums[i] != 0):
                nums[i], nums[count] = nums[count], nums[i]
                count = count+1